src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,184 @@
+                            # except for the addition of the 'id' field
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "",
+                        "description": "Wrong Value",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Wrong Value"
+                    }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                        "$.codeValue": [
+                            "CodeValue is required and should not be left empty."
+                        ]
+                  }
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data Forbidden
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "",
+                        "description": "Wrong Value",
+                        "namespace": "uri://.org/wrong",
+                        "shortDescription": "Wrong Value"
+                    }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                        "$.codeValue": [
+                                "CodeValue is required and should not be left empty."
+                            ]
+                        }
+                    }
+                  """
+        @ignore
+        Scenario: Verify error handling with POST using empty body
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                    }
+                  """
+             Then it should respond with 403
```

Comment:
Shouldn't this be a 400 response rather than 403?

Reply:
Typo, fixed...

Reply:
Still needs to be 400 instead of 403.